source-control/pull-request-templates/README.md

```diff
@@ -0,0 +1,47 @@
+# Pull Request Templates
```

Comment:
Since this is a new section to the _source control_ section of the docs, we need to link over to it from [here](
https://github.com/microsoft/code-with-engineering-playbook/blob/master/source-control/readme.md).